src/subscript/config_jobs/CASEGEN_UPCARS

```diff
@@ -0,0 +1,10 @@
+EXECUTABLE    casegen_upcars
+
+ARG_TYPE 0 STRING
+ARG_TYPE 1 STRING
+ARG_TYPE 2 STRING
+
+ARGLIST         <CONFIG>  "--et" <ECLTMP>  "--base" <ECL> 
+
```

Comment:
Are `<ECLTMP>` and `<ECL>` representing input and output data files?
If yes, should we consider using other names for `<ECLTMP>` and `<ECL>`?

> Some random suggestions
>  ECLTMP --> `<INPUT>`, `<ECLIPSE_TEMPLATE>`  or similar 
>  ECL --> `<OUTPUT>`, `<ECLIPSE_OUTPUT>` ??
>

Reply:
Agree and I like your suggestions :+1: